faq_bot/.env

```diff
@@ -0,0 +1 @@
+TG_BOT_TOKEN=bot_token
```

Comment:
Now we can add `faq/.env` line into `.gitignore` to assist in making of changes for this file

Reply:
https://github.com/deeppavlov/dialog_flow_demo/pull/1/commits/e6d1dd4067811f0eef73c8b5665542fdaf97bb19